numpy/linalg/cholesky.cc

```diff
@@ -45,50 +30,86 @@ void Cholesky::make_args(int size) {
     for (int i = 0; i < n; i++) {
         r_mat[i * n + i] = 1;
     }
-    cblas_dgemm(CblasColMajor, CblasNoTrans, CblasTrans, n, n, n, 1.0, x_mat, n,
-                x_mat, n, n, r_mat, n);
+
+    static const char uplo = 'U';
+    static const char trans = 'T';
+    static const double alpha = 1.;
+    static const double beta = n;
+    dsyrk(&uplo, &trans, &n, &n, &alpha, x_mat, &n, &beta, r_mat, &n);
```

Comment:
It may speed set-up up significantly if we were to allow `x_mat` to be non-square, but already using `dsyrk` vs. `dgemm` should give factor of 2 speed improvement.